app/controllers/api/v2/search_controller.rb

```diff
@@ -24,6 +28,49 @@ def matching_suggestions
 
         []
       end
+
+      def log_search_results(query, results)
```

Comment:
Is it worth extracting this into a service?

Reply:
I thought about putting this through ActiveSupport::Notifications. I'll look into that again

Reply:
Moved to a service now